src/formatDate.js

```diff
@@ -50,7 +50,36 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
+
+  const fromDateObject = {};
+  const toDateArray = [];
+
+  let index = 0;
+
+  for (let i = 0; i <= 2; i++) {
```

Comment:
I think you can replace '2' with 'fromFormat.length - 1' , because it looks like a magic number:)

Reply:
agree